lib/plugins/entities.js

```diff
@@ -65,6 +65,20 @@ function inject (bot, { version }) {
   bot.uuidToUsername = {}
   bot.entities = {}
 
+  bot.nearestEntity = (match = (entity) => { return true }) => {
+    let best = null
+    let bestDistance = Number.MAX_VALUE
+    for (const entity of Object.values(bot.entities)) {
+      if (entity === bot.entity || !match(entity)) continue
+      const dist = bot.entity.position.distanceSquared(entity.position)
+      if (dist < bestDistance) {
+        best = entity
+        bestDistance = dist
+      }
+    }
+    return best
```

Comment:
I'd only add a couple of newlines to make it more readable and add brackets to L72. Other than that, LGTM

Reply:
done

Reply:
It's completely fine both ways but I don't find adding these empty lines more readable, more the opposite actually since it takes more room and requires more scrolling.

Reply:
I tend to write more compact code for the reasons rom cited. But mineflayer is also aimed at being read by beginners, so maybe it is more readable for them ? Idk, its up to you